_targets/build.project.armv7m7-imxrt105x

```diff
@@ -55,13 +55,15 @@ KERNEL_PHBASE=$((0x0)) # itcm start address
 KERNEL_PHOFFS=$((0x0))
 export KERNEL_PHADDR=$(printf "%08x" $((KERNEL_PHBASE + KERNEL_PHOFFS)))
 
+# All data/code maps need to be cacheable to allow unaligned accesses
 
 # Pre-init script is launched before user script
 PREINIT_SCRIPT=(
 	"map itcm 0x0 0x28000 rwx"
```

Comment:
Shouldn't this be `rwxcb`?
```suggestion
	"map itcm 0x0 0x28000 rwxcb"
```

Reply:
From my testing it doesn't seem to do anything anyway, but you're right, will remove these attributes from tcms

Reply:
Hmm, on the other hand, if I remember correctly, it's needed anyway for unaligned access to work. I'll run the CI one last time

Reply:
Yup, it crashes, I'm restoring cacheable flags

Reply:
@gerard5 these flags do not change anything about behaviour of TCM, but if they're absent, unaligned access with cause usage fault (unaligned access is disallowed on strongly ordered memory, regardless of SCB config, it's specified int the cortex-m manual)